pandas/tests/arithmetic/test_numeric.py

```diff
@@ -405,6 +406,11 @@ def test_ser_div_ser(self, dtype1, any_real_dtype):
                 name=None,
             )
         expected.iloc[0:3] = np.inf
+        if first.dtype == "int64" and second.dtype == "float32":
```

Comment:
is the reverse excluded as well?

Reply:
The reverse (float32 + int64) is not tested, as the `first.dtype` is always int64/float64/uint64